django_project/frontend/src/pages/Admin/Basemap/List/index.jsx

```diff
@@ -34,8 +34,15 @@ export function resourceActions(params) {
  */
 export default function BasemapList() {
   const pageName = pageNames.Basemaps
-  const columns = COLUMNS(pageName, urls.admin.basemapList);
-  columns[4] = {
+  let columns = COLUMNS(pageName, urls.admin.basemapList);
+  columns.pop();
+  columns = columns.concat([
+    { field: 'created_at', headerName: 'Created At', flex: 0.5, type: 'date' },
```

Comment:
I need to put these fields into 1 usable variable. Any suggestion on where I should put it? @meomancer

Reply:
You could put here @zamuzakki 
https://github.com/unicef-drp/GeoSight-OS/blob/4dc4e12155ab65ad1e058dbfdf6867c8b62ff9fc/django_project/frontend/src/components/AdminList/types.d.ts